backend/src/apiserver/client_manager.go

```diff
@@ -153,6 +154,7 @@ func (c *ClientManager) Authenticators() []auth.Authenticator {
 func (c *ClientManager) init() {
 	glog.Info("Initializing client manager")
 	db := initDBClient(common.GetDurationConfig(initConnectionTimeout))
+	db.SetConnMaxLifetime(time.Second * time.Duration(common.GetIntConfigWithDefault(dbConMaxLifetimeSec, 120)))
```

Comment:
Should we follow the way of how we set the default value of initConnectionTimeout  https://github.com/kubeflow/pipelines/blob/6717434978f9c9d01d4c38914dc3e858a6a6c2fd/backend/src/apiserver/config/config.json#L16 instead of setting the default value `120` in the code.

Reply:
Certainly, I think my latest commit covers this concern as well as documentation and deployment.